ONSdigital/dp-frontend-renderer
# Added language toggle to CMD pages

### What

Changed text "Change language: English (EN) " and "Change language | Cymraeg (CY)" to "English (EN) | Cymraeg (CY)"

### How to review

1. Checkout PR
2. Go into responsive move width <767
3. Open menu, check language is present
4. Make viewport larger, ensure normal change language and only that change language option is available.
Check the text reads "English (EN) | Cymraeg (CY)"

### Who can review

Anyone except me
